app/src/main/java/com/nickytoolchick/agraph/ui/ChartOptionsActivity.kt

```diff
@@ -27,7 +27,7 @@ class ChartOptionsActivity : AppCompatActivity() {
 
         binding.submitChartOptionsButton.setOnClickListener {
             updateChartOptions()
-            mainActivityIntent.putExtra("result", Json.encodeToString(chartOptions))
+            mainActivityIntent.putExtra("newChartOptions", Json.encodeToString(chartOptions))
```

Comment:
Do not use hardcoded identifiers.

Transferring an extra data identifier to a separate Enum class or object can be a good practice. This can ensure the correctness of the identifier and help avoid typos or errors in the use of identifiers for the correct operation of the program

* Example:
```kotlin
object Exstras{
val NEW_CHART_OPTIONS = "newChartOptions"
}
```